docs/ja/docs/advanced/websockets.md

```diff
@@ -0,0 +1,172 @@
+# WebSocket
+
+**FastAPI**で<a href="https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API" class="external-link" target="_blank">WebSocket</a>が使用できます。
+
```

Comment:
It seems to be added install section in English after created this pull request. 

https://github.com/tiangolo/fastapi/commit/ea017c99198c73f0e4a67a758bc5113e2cca6102

````suggestion

## `WebSockets`のインストール

まず `WebSockets`のインストールが必要です。

<div class="termy">

```console
$ pip install websockets

---> 100%
```

</div>

````